web-admin/src/components/home/DashboardList.svelte

```diff
@@ -16,6 +16,13 @@
   }
 
   async function updateDashboardsForProject() {
+    const status = $proj.data?.prodDeployment.status;
+    if (
+      status === "DEPLOYMENT_STATUS_PENDING" ||
```

Comment:
These shouldn't be hardcoded strings, but rather e.g. `V1DeploymentStatus.DEPLOYMENT_STATUS_RECONCILING`